[item] Missing `types` in package.json

While preparing `@xethya/inventory` to use `@xethya/item`, the following error occured:

```
    tests/inventory.test.ts:1:22 - error TS7016: Could not find a declaration file for module '@xethya/item'. '/home/joel/Projects/xethya/framework/node_modules/@xethya/item/dist/xethya.item.js' implicitly has an 'any' type.
      Try `npm install @types/xethya__item` if it exists or add a new declaration (.d.ts) file containing `declare module '@xethya/item';`

    1 import { Item } from "@xethya/item";
                           ~~~~~~~~~~~~~~
```

--- a/packages/item/package.json
+++ b/packages/item/package.json
@@ -1,11 +1,12 @@
 {
   "name": "@xethya/item",
   "shortName": "item",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "",
   "main": "dist/xethya.item.js",
   "module": "dist/xethya.item.es.js",
   "iife": "dist/xethya.item.iife.js",
+  "types": "dist/index.d.ts",
   "repository": "https://github.com/xethya/framework",
   "author": "Joel A. Villarreal Bertoldi",
   "license": "MIT",

